_docs/rfc/TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+# RFC Template
```

Comment:
Is there a link leading to the RFC template? If not, consider adding one in `CONTRIBUTING.md`

Reply:
Yes! It's on line 28.